### DISCUSSION TITLE
Deletes 9-12 messages and stops

### DISCUSSION THREAD

**Discussion by tanyamga** [victornpb/undiscord#500] [1. Questions & Answers]
I want to delete all the messages ive sent with my discord account but the bot stops at around 9-12 messages deleted for no reason. i have thousands of messages i want gone so this is really inconvenient! i would appreciate any help with this

**Comment by 573rn** [comment]
Set search delay to something like 14000 - 25000

It's slow, but it gives the bot enough time to cache the next page.
